sql/log_event.cc

```diff
@@ -9681,6 +9681,19 @@ search_key_in_table(TABLE *table, MY_BITMAP *bi_cols, uint key_type)
   KEY *keyinfo;
   uint res= MAX_KEY;
   uint key;
+  /*
+    PK has bugs, not support
+  */
+  bool tianmu_engine = table && table->s && 
+                      (table->s->db_type() ? table->s->db_type()->db_type == DB_TYPE_TIANMU: false);
+  enum_sql_command sqlCommand = SQLCOM_END;
```

Comment:
Definition the sql command variable in this form:
```
enum_sql_command save_sql_command= lex->sql_command;
```
The SQLCOM_END should not be an init value.